atd-cr3-api/server.py

```diff
@@ -425,42 +425,54 @@ def user_delete_user(id):
         abort(403)
 
 
-@APP.route("/crashes/associate_location/", methods=["PUT", "POST"])
+@APP.route("/events/", methods=["PUT", "POST"])
 def associate_location():
     # Require matching token
-    incoming_token = request.headers.get('Hasura-Event-Api')
+    incoming_token = request.headers.get("HASURA_EVENT_API")
+    incoming_event_name = request.headers.get("HASURA_EVENT_NAME", "")
     hashed_events_api = hashlib.md5()
     hashed_events_api.update(str(HASURA_EVENT_API).encode("utf-8"))
     hashed_incoming_token = hashlib.md5()
     hashed_incoming_token.update(str(incoming_token).encode("utf-8"))
 
     # Return error if token doesn't match
     if hashed_events_api.hexdigest() != hashed_incoming_token.hexdigest():
-        return {
-            "statusCode": 403,
-            "body": json.dumps({'message': 'Forbidden Request'})
-        }
+        return {"statusCode": 403, "body": json.dumps({"message": "Forbidden Request"})}
+
+    # Check if there is an event name provided, if not provide feedback.
+    if incoming_event_name == "":
+        return {"statusCode": 403, "body": json.dumps({"message": "Forbidden Request"})}
```

Comment:
Should this error message include that the event name is missing? Just to be more descriptive for troubleshooting?

Reply:
Good call, just modified it 👍